jupyterlab_bigquery/src/components/details_panel/table_details_panel.tsx

```diff
@@ -95,7 +95,19 @@ export default class TableDetailsPanel extends React.Component<Props, State> {
             Data location: {details.location ? details.location : 'None'}
           </div>
           <br />
-          <div>Schema: {details.schema ? details.schema : 'None'}</div>
+          <div>
+            Schema:{' '}
+            {details.schema
+              ? details.schema.map((field, index) => {
+                  return (
+                    <div key={index}>
+                      {field.name} {field.type} {field.mode}{' '}
```

Comment:
What is the purpose of the {' '}?

Reply:
it's just a placeholder space that got inserted because the line wrapped around. Ultimately the schema won't look like a plain string like this, but this PR is just an intermediate step before another styling PR.

Reply:
Sounds good